bin/create-ebs-volume

```diff
@@ -149,10 +149,11 @@ function get_next_logical_device() {
     for letter in ${alphabet[@]}; do
         # use /dev/xvdb* device names to avoid contention for /dev/sd* and /dev/xvda names
         # only supported by HVM instances
-        if [ ! -b "/dev/xvdb${letter}" ]; then
+        if [[ $created_volumes =~ .*/dev/xvdb${letter}.* ]]; then
```

Comment:
Is the `.*` at the start of the pattern needed, and can the `.*` at the end of the pattern be scoped to a subset - e.g. `[0-9]`?

Reply:
The string in created_volumes comes from $(aws ec2 describe-volumes) which is JSON output, to avoid having to actually parse it (and pull in jq or python or something) it just does the simpler match to see if "/dev/xvdb${letter}" shows up anywhere in the describe-volumes output, which is good enough to guess that we probably shouldn't try to use it.

Reply:
So the leading and trailing `.*` are there to consume all the other stuff we don't care about.